src/TasksApi.ts

```diff
@@ -0,0 +1,34 @@
+import type { App } from 'obsidian';
+import type { Task } from './Task';
+import { TaskModal } from './TaskModal';
+import { taskFromLine } from './Commands/CreateOrEditTaskParser';
+
+export class TasksApi {
```

Comment:
With a mind to being able to change the interface of the Api in future, I had in mind to call the first API 'V1' - so perhaps `TasksApiV1`.

Reply:
Versioning is a good point. Will change class name to `TasksApiV1` and the method to access the API to `apiV1()`.
Is this ok?

Reply:
> Versioning is a good point. Will change class name to `TasksApiV1` and the method to access the API to `apiV1()`.
> Is this ok?

Perfect. (I had forgotten about the access method)